README.adoc

```diff
@@ -79,29 +81,30 @@ Assuming apps have been deployed configuring `spring.cloud.stream.bindings.appli
 
 === Controlling eviction
 
-Internally the collector maintains a cache of the metrics it receives. The default for metric emission is every 5 seconds, but can be tuned on the application by using Spring Boot's metrics exporter scheduling control, please refer to the docs http://docs.spring.io/spring-cloud-stream/docs/Chelsea.SR1/reference/htmlsingle/index.html#_metrics_emitter[here] to configure your applications.
+Internally the collector maintains a cache of the metrics it receives. The default for metric emission is every 60 seconds (for Spring Boot 1.x applications it is every 5 seconds), but can be tuned on the application by using Spring Boot's metrics exporter scheduling control, please refer to the docs https://docs.spring.io/spring-cloud-stream/docs/Elmhurst.RELEASE/reference/htmlsingle/index.html#spring-cloud-stream-overview-metrics-emitter[here] to configure your applications.
```

Comment:
Does it mean, we will see throughput rate changes only after every 60s in SCDF UI? Or, is it about the cache TTL?

Reply:
Yes!
If the metrics emitted from the SCSt apps come every 60 sec. (this is the default so SCSt2.x) then the UI will be able to show changes at 60 sec rate. 
The important bit is to align the MC's cache retention interval to be bigger then the default metric emission interval. That is the reason we have increased it from 30 to 90 seconds.  If the MC's cache retention time is smaller then the emission time then the UI will experience disappearance and reappearance of the metrics visualization.

Note that all the time intervals can be changed at deployement. 
- for emission time interval: `spring.cloud.stream.metrics. scheduleInterval`
- for MC retention time: `spring.cloud.dataflow.metrics.collector.evictionTimeout`